### ISSUE TITLE
Prefetch row plural name for included table ending with numeral

### ISSUE THREAD

**Issue by martindufort** [groue/GRDB.swift#1041]
The SQLAssociationStep struct is pluralizing `focus_email_v2` into `focus_email_vs2`.
I was expecting the plural to be `focus_email_v2s`. Is this the proper expectation?

I'm using this key to test for the availability of specific prefetched rows.

```
// Only decode focus emails if they were prefetched
if row.prefetchedRows.keys.contains("focus_emails_v2s") {
	self.attachedEmails = row["focus_emails_v2s"]
} 
else {
	self.attachedEmails = nil
}
```

```
struct SQLAssociationStep: KeyPathRefining {
    var key: SQLAssociationKey
    var condition: SQLAssociationCondition
    var relation: SQLRelation
    var cardinality: SQLAssociationCardinality
    
    var keyName: String {
==>        return key.name(for: cardinality)
    }
}
```

Happening in 
```
Inflections.swift: 23
```

I would gladly send a pull request but I'm always at lost with Regular Expressions.

**Comment by groue** [comment]
Hello @martindufort,

Your solution is to give a custom key to your association:

```swift
let myAssoc = hasMany(...).forKey("focus_email_v2s")
//                        ~~~~~~~~~~~~~~~~~~~~~~~~~~
```

---

Inflection rules can not be modified. This would be too hard a breaking change. Associations are already complicated enough. People would just not know why their code starts misbehaving and how to fix it. Practically speaking, inflection rules are engraved _forever_. Even future major releases will not change them.

In order to mitigate pain points, I imported them from the excellent Ruby on Rails, and I also looked for reported (and unfixable) RoR issues so that those would not ship into GRDB.

It happens that I also added the numeric suffix extraction because the use case I had in mind was `player1`, `player2`, ... (I did not think of `player_v2`). That's why `player1` is  pluralized into `players1`.

This behavior is [hard-coded](https://github.com/groue/GRDB.swift/blob/v5.10.0/GRDB/Utils/Inflections.swift#L191), and not fixable even with regular expression skills.
